Homework/2/Program.cs

```diff
@@ -0,0 +1,390 @@
+        static void MenuSelection() {
+            Console.Clear();
+
+            if (loggedIn)
+            {
+                Console.WriteLine("Logged in. \n\n");
+                Console.WriteLine("3 - List users\n");
+                Console.WriteLine("4 - Add user to start of list\n");
+                Console.WriteLine("5 - Add user to end of list\n");
+                Console.WriteLine("6 - Add user to n'th position in list\n");
+                Console.WriteLine("7 - Remove user from start of list\n");
+                Console.WriteLine("8 - Remove user from end of list\n");
+                Console.WriteLine("9 - Remove n'th user\n");
+                Console.WriteLine("10 - Sort users by username (asc/dec)\n");
+                Console.WriteLine("11 - Log out\n");
+            }
+            else {
+                Console.WriteLine("Select option:\n");
+                Console.WriteLine("1 - Log in\n");
+                Console.WriteLine("2 - Register\n");
+            }
+
+            OptionSelection(option: Console.ReadLine());
+        }
+
+        //Funtion that gets which option was selected and calls other needed functions via switch statement.
+        static void OptionSelection(string option) {
+            Console.Clear();
+            switch (option)
+            {
+                case "1":
+                    // Log in
+                    Login();
+                    break;
+                case "2":
+                    //Register / adding to end of list
+                    PromptAddUserToArray();
+                    break;
+                case "3":
+                    //Listing all users;
+                    ListUsers();
+                    break;
+                case "4":
+                    //Adding to start of list
+                    PromptAddUserToArray(addToEnd: false, addToStart: true);
+                    break;
+                case "5":
+                    //adding to end of list/ like register
+                    PromptAddUserToArray();
+                    break;
+                case "6":
+                    //Adding to n'th(index) position on list
+                    PromptAddUserToArray(addToEnd: false, addToStart: false);
+                    break;
+                case "7":
+                    //Remove from start of the array
+                    PromptRemoveUser(removeFromEnd: false, removeFromStart: true);
+                    break;
+                case "8":
+                    //Remove from end of the array
+                    PromptRemoveUser(removeFromEnd: true, removeFromStart: false);
+                    break;
+                case "9":
+                    //Prompt to get index of user to remove from array
+                    PromptRemoveUser(removeFromEnd: false, removeFromStart: false);
+                    break;
+                case "10":
+                    //Sorting array by usernames in ascending order
+                    SortUsers();
```

Comment:
Do those comments add any value? They are awfully similar to the method names. Maybe you could clarify method names instead?